indexer/map_object.cpp

```diff
@@ -188,6 +190,33 @@ string MapObject::GetBuildingLevels() const
   return m_metadata.Get(feature::Metadata::FMD_BUILDING_LEVELS);
 }
 
+vector<string> MapObject::GetBusRoutes() const
+{
```

Comment:
These three methods are almost the same (the only difference is the metadata tag). Could you please create a single method in the anonymous namespace, like:
```c++
vector<string> GetRoutes(Metadata & md, Metadata::EType type)
{
  ...
}
```

and then use it in three places?